subseas_flex_fcst/config-sample.yaml

```diff
@@ -11,13 +11,9 @@ loglevel: info
 
 # Forecast options
 y_transform: true
-forecast_mu_file: CFSv2_SubXPRCP_CCAFCST_mu_Apr_Apr-1-2022_wk1.txt
```

Comment:
Note that you need one file to read for the layout initialization

Reply:
Ah yes I did not notice that.